assets/src/stories-editor/blocks/amp-story-text/edit.js

```diff
@@ -74,6 +75,12 @@ class TextBlockEdit extends Component {
 			prevProps.attributes.content !== content
 		);
 
+		// If block has been pasted, then regenerate height and width.
+		if ( isPasted ) {
+			setAttributes( { isPasted: false } );
+			maybeUpdateBlockDimensions( this.props );
+		}
+
```

Comment:
Should `if ( checkBlockDimensions ) {` be with `else` now? To ensure it's not happening the second time after it's already done with `isPasted`? Not sure if that would ever be the case though.

Reply:
We could do this 

```javascript
if ( checkBlockDimensions || isPasted ) {
	maybeUpdateBlockDimensions( this.props );
}

if ( isPasted ) {
	setAttributes( { isPasted: false } );
}
```

Reply:
👍 
Alternatively, we could add `isPasted` to the `checkBlockDimension` conditions so that if  `isPasted`, `checkBlockDimensions` would be `true`. And then just add the `if ( isPasted )` for setting it `false` as in your previous comment.

Reply:
@miina's latter suggestion also "fixes" that case that stuff pasted into a fitted text box doesn't require block dimension updates. But that doesn't actually seem to be a problem anyway.

Reply:
The above code snippet feels the safest, as it is much more human readable here. I will update my PR.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3347/commits/4460c2e171e129a41b20d6a9cb475d29cf2bae64